### DISCUSSION TITLE
群全部已满，没人处理么

### DISCUSSION THREAD

**Discussion by ruixiaozi** [NervJS/taro#13787] [生态]
群全部已满，没人处理么

**Comment by TheKonka** [comment]
好像是因为企业微信的相关限制
![image](https://user-images.githubusercontent.com/22173084/236628984-0bd0181b-8456-4770-9734-13f6e6983385.png)
有个开发交流群54满200人了，可以加我微信我拉你进去
另外，不要重复发discussions刷屏

**Reply by honlyHuang** [reply]
已经修复
